community/modules/scheduler/schedmd-slurm-gcp-v6-controller/variables.tf

```diff
@@ -104,6 +104,19 @@ variable "login_nodes" {
       auto_delete  = optional(bool, true)
       boot         = optional(bool, false)
     })), [])
+    additional_networks = optional(list(object({
```

Comment:
Could you please match the definition with one from `community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf : L404`. Those extra fields will remain inert (e.g. `queue_count`), but we would prefer to "over-provision". Thank you.

Reply:
Thanks for the pointer. Added.